.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] [OPTIONAL] Nice work creating the `.gitignore` file and adding the `node_modules/` folder to it however, according to the project's [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you need to add the `test.md` file to it as well, Please makes sure to follow all the requirements in all your projects before submitting for review and keep up the good work 👍🏽